--- ../../fuse-0-branch-2.2/example/fusexmp.c	2005-11-17 16:29:01.000000000 +0100
+++ fusexmp.c	2005-03-27 15:48:53.000000000 +0200
@@ -73,7 +73,12 @@
 {
     int res;
 
-    res = mknod(path, mode, rdev);
+    /* On the Hurd we must not use mknod() to create files, but creat() */
+    if(mode & S_IFREG)
+      res = creat(path, mode & ALLPERMS);
+    else
+      res = mknod(path, mode, rdev);
+
     if(res == -1)
         return -errno;
 
